packages/db-schema/src/voting/defs.ts

```diff
@@ -183,31 +171,84 @@ export const boostBalances = pgTable(
 export const boostChanges = pgTable(
   "boost_changes",
   {
-    id: uuid("id").primaryKey().notNull(),
+    id: uuid("id").primaryKey().notNull().defaultRandom(),
     balanceId: uuid("balance_id")
       .notNull()
       .references(() => boostBalances.id, {
         onDelete: "restrict",
         onUpdate: "cascade",
       }),
-    deltaAmount: bigint("delta_amount", { mode: "bigint" }).notNull(), // earn:+X, spend:-X
+    wallet: bytea("wallet").notNull(),
```

Comment:
Always track the wallet with each change event. These are connected to the user via the `balanceId` column. One day, we might make a table to hold wallet addresses per user, then reference that table from this one to save some bytes, but for now this is fine.